frontend/src/components/context_menu/context_menu.vue

```diff
@@ -496,6 +537,56 @@
           </v-list-item-title>
         </v-list-item-content>
       </v-list-item>
+      <v-list-item
+        link
+        dense
+        data-cy="delete_instance"
```

Comment:
may wish to rename these data-cy elements on merge and unmerge buttons

Reply:
Thanks @anthony-sarkis ! I just pushed the fix for this :)